spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/deployer/strategies/SimpleRedBlackUpgradeStrategy.java

```diff
@@ -53,66 +48,42 @@
 
 	private final AppDeploymentRequestFactory appDeploymentRequestFactory;
 
-	private final HealthCheckAndDeleteStep healthCheckAndDeleteStep;
+	private final HealthCheckStep healthCheckStep;
+
+	private final HandleHealthCheckStep handleHealthCheckStep;
+
+	private final ReleaseRepository releaseRepository;
+
+	private final DeployAppStep deployAppStep;
 
 	public SimpleRedBlackUpgradeStrategy(DeployerRepository deployerRepository,
```

Comment:
After simplification, there are dependencies such as `deployerRepository`, `appDeployerDataRepository`, `appDeploymentRequestFactory` and `releaseRepository` are not needed in this class.

Reply:
can you remove them on merge?